04-store-api/starter/db/connect.js

```diff
@@ -1,12 +1,7 @@
-const mongoose = require('mongoose')
+const mongoose = require("mongoose");
 
 const connectDB = (url) => {
-  return mongoose.connect(url, {
```

Comment:
In order to use `await` on this inside `app.js` you need to return the call to `mongoose.connect()`. I wouldn't do away with this change!

Reply:
For context, this will ensure that you return a promise back to the function that calls `connectDB()`.